crates/daphne/src/protocol/mod.rs

```diff
@@ -208,113 +198,6 @@ mod test {
 
     test_versions! { roundtrip_report }
 
-    #[test]
```

Comment:
AFAICT the only difference between this test and the previous is that the `prio_draft09` crate is used instead of `prio`. This was necessary because we needed the right version of Prio3Sum. We avoid this by using a VDAF for testing that is supported by both versions (Prio2).